### DISCUSSION TITLE
[Base] Get multiple keys

### DISCUSSION THREAD

**Discussion by Roosteridk** [deta/cloud-docs#710] [Ideas]
Currently you can only get keys individually. Being able to get multiple items would reduce number of requests when there is a large list of items.

**Comment by sudo-subg** [comment]
As of now, Deta Base has bunch of basic features, but they have plans for adding more features to Deta Base in the future (more like Deta-Base v2). Until then you can make asynchronous get requests as a workaround for getting multiple keys. There are bunch of third-party SDKs which has this functionality.

**Reply by Roosteridk** [reply]
I know you can just send out multiple requests but it would be nice to be able to do it in a single request :)
